sonic_data_client/virtual_db.go

```diff
@@ -166,7 +166,14 @@ func getPfcwdMap() (map[string]map[string]string, error) {
 			log.V(1).Infof("PFC WD not enabled on device")
 			return nil, nil
 		}
-		qos_key := resp[0]
+
+		var qos_key string
+		for _, key := range resp {
+			if strings.Contains(key, "Ethernet") { // Account for PORT_QOS_MAP|global
```

Comment:
>global [](http://example.com/codeflow?start=70&length=6)

could you just check "global" in the loop?

Reply:
I understand this is to filter all key containing "Ethernet". However this prefix may be changed in future. Check not equal to "global" may be safer.